src/lib/namespaces/state.ts

```diff
@@ -288,4 +288,23 @@ export default class State extends Namespace {
 				});
 		});
 	}
+
+	/**
+	 * Get Tip of the Block on the node.
```

Comment:
This comment isn't correct. Should be `Return the block with the highest height or number on the node`

Reply:
ok seen